test/Connection.js

```diff
@@ -2,7 +2,6 @@ import nock from 'nock'
 import fs from 'fs'
 import path from 'path'
 import { expect } from 'chai'
-import XHRMock from 'xhr-mock'
```

Comment:
I removed `XHRMock` because it doesn't work with the latest `axios` implementation.
It was used to test the behavior of `onUploadProgress` and `onDownloadProgress`, but I think it is sufficient if we test the options are passed to `axios` correctly.